meta-resin-common/recipes-bsp/u-boot/patches/resin-specific-env-configuration.patch

```diff
@@ -52,17 +58,33 @@ index 0000000..ef8a2b0
 +		"echo Import ${resin_env_file} in environment;" \
 +		"env import -t ${resin_kernel_load_addr} ${filesize}\0" \
 +	\
++	"resin_flasher_detect=" \
++		"fatload mmc " __stringify(RESIN_EXTERNAL_MMC) ":${resin_boot_part} ${resin_kernel_load_addr} ${resin_flasher_external_flag_file};\0" \
++	\
++	"resin_set_mmcdev=" \
++		"if test -n " __stringify(RESIN_EXTERNAL_MMC) "; then " \
++			"if run resin_flasher_detect; then " \
++				"setenv resin_mmcdev " __stringify(RESIN_EXTERNAL_MMC) "; "\
++			"else; " \
++				"setenv resin_mmcdev " __stringify(RESIN_INTERNAL_MMC) "; "\
++			"fi; " \
++		"else; "\
++			"setenv resin_mmcdev " __stringify(RESIN_INTERNAL_MMC) "; "\
++		"fi;\0" \
++	\
 +	"resin_inject_env_file=" \
 +		"if run resin_load_env_file; then " \
 +			"run resin_import_env_file;" \
 +		"fi;\0" \
 +	\
 +	"resin_set_kernel_root=" \
++		"run resin_set_mmcdev;" \
 +		"run resin_inject_env_file;" \
```

Comment:
Can you just do a "run resin_load_env_file" here? Because I don't see "resin_import_env_file" defined anywhere